contrib/mkimage.sh

```diff
@@ -71,18 +86,18 @@ nameserver 8.8.8.8
 nameserver 8.8.4.4
 EOF
 
-tarFile="$dir/rootfs.tar.xz"
+tarFile="$dir/rootfs.tar${compression}"
 touch "$tarFile"
 
 (
 	set -x
-	tar --numeric-owner -caf "$tarFile" -C "$rootfsDir" --transform='s,^./,,' .
+	tar --numeric-owner --create --auto-compress --file "$tarFile" --directory "$rootfsDir" --transform='s,^./,,' .
 )
 
 echo >&2 "+ cat > '$dir/Dockerfile'"
-cat > "$dir/Dockerfile" <<'EOF'
+cat > "$dir/Dockerfile" <<EOF
 FROM scratch
-ADD rootfs.tar.xz /
+ADD $tarFile /
```

Comment:
This actually needs to be `$(basename "$tarFile")` since we don't want the directory name on the front (and `$tarFile` has that).